index.js

```diff
@@ -1,7 +1,54 @@
 // Iteration 1: Names and Input
 
+var hacker1 = "John";
+
+var hacker2 = "lawrence";
+
+console.log("The navigator's name  ${hacker2}");
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log(
+    "The driver has the longest name, it has ${hacker1.length} characters."
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    "It seems that the navigator has the longest name, it has ${hacker2.length} characters."
+  );
+} else {
+  console.log(
+    "Wow, you both have equally long names, ${hacker1.length} characters!"
+  );
+}
 
 // Iteration 3: Loops
+
+// 3.1
+var captaliseName = hacker1.toUpperCase();
+
+var splitName = captaliseName.split(" ").join(" ");
```

Comment:
This is incorrect. `splitName === capitaliseName`. It shouldn't be. Check the split function for the bug.

Reply:
This has been completed